Source/WebCore/rendering/ReferencedSVGResources.h

```diff
@@ -74,6 +74,7 @@ class ReferencedSVGResources {
     static RefPtr<SVGClipPathElement> referencedClipPathElement(TreeScope&, const ReferencePathOperation&);
     static RefPtr<SVGMarkerElement> referencedMarkerElement(TreeScope&, const String&);
     static RefPtr<SVGMaskElement> referencedMaskElement(TreeScope&, const StyleImage&);
+    static RefPtr<SVGMaskElement> referencedMaskElement(TreeScope&, const AtomString& fragment);
```

Comment:
You can omit the parameter description, no?

Reply:
Done.